en/docs/administration/configure-custom-domain.md

```diff
@@ -0,0 +1,27 @@
+# Configure Custom Domain
+
+A custom domain is essential for proper branding, discoverability, and credibility of your site. Choreo allows you to configure a custom domain for your organization's developer portal.
+
+Follow the steps below to configure the custom domain for the developer portal:
+
+## Prerequisites
+
+1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
+2. If you are a new user, create an organization. Enter a unique organization name and create an organization. For example, "Stark Industries".
+    
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+3. Obtain a custom domain through a domain registrar. 
+4. Through your DNS administrator, create a `CNAME` type DNS record and associate it with the Choreo custom developer access domain: `customdns.devportal.choreo.dev`. 
```

Comment:
Should we change "Choreo custom developer access domain" to "Choreo Developer Portal custom domain"?

Reply:
done